README.md

```diff
@@ -0,0 +1,42 @@
+# Introduction
```

Comment:
Add our CI badge at the top:
```md
[![pipeline status](https://gitlab.com/ultimaker/embedded/platform/um-update_toolbox_armhf/badges/master/pipeline.svg)](https://gitlab.com/ultimaker/embedded/platform/um-update_toolbox_armhf/commits/master)

```

Reply:
So not add it..?

Reply:
yeah I don't think we should, keep the documentation clean (otherwise you have annoyances like what branch do we refer to in the document etc)